pkg/containerd/cmd/service/skanky-vendor.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+#
+# We only need the containerd client and its transitive dependencies
+# and we conveniently have a checkout already. We actually prefer to
+# reuse containerd's vendoring for consistency anyway.
+
+set -eu
+ctrd=$1
+mkdir -p vendor/github.com/containerd/containerd
+ln -nfs $ctrd/* vendor/github.com/containerd/containerd
+rm -f vendor/github.com/containerd/containerd/vendor
+
+for p in github.com/Microsoft/go-winio \
```

Comment:
I don't think it should be any more difficult to maintain than a `vendor.conf`, either as the client code here changes or as the `CONTAINERD_COMMIT=` in the `Dockerfile` changes -- which is roughly equivalent in this context to changing the containerd entry in a `vendor.conf` and then syncing all the other entries from containerd's `vendor.conf`, which is how I would do things if I was doing this with a `vendor.conf`.